scripts/chart/index

```diff
@@ -12,4 +12,4 @@ repo_index=${1}
 echo "Getting current index from ${repo_index}"
 curl -f -H 'Cache-Control: max-age=0,no-cache' -H 'Host: releases.rancher.com' "https://c.storage.googleapis.com/server-charts/${repo_index}/index.yaml?$(date +%s%N)" -o build/index.yaml
 
-helm repo index --merge build/index.yaml ./bin/chart/${repo_index}
+helm_v3 repo index --merge build/index.yaml ./bin/chart/${repo_index}
```

Comment:
Just to clarify, the reason why you are suggesting this change is to ensure that helm_v3 is used instead of helm, which points to helm v2 based on the following code, right?

https://github.com/rancher/rancher/blob/51d074b202ee3445bb87b827fc1f06f5a66627ff/Dockerfile.dapper#L55-L68